doc/build-windows.md

```diff
@@ -101,6 +101,10 @@ Build using:
     CONFIG_SITE=$PWD/depends/x86_64-w64-mingw32/share/config.site ./configure --prefix=/
     make
 
+## Building using a native compiler tool chain such as Visual Studio
+
+Full instructions are available in [README.md](..//build_msvc/README.md).
```

Comment:
This is already mentioned in the bullet point in the first section

> On Windows, using a native compiler tool chain such as Visual Studio.

Please just add the link there

Reply:
Yes this is better.
new commit added